app/templates/index.html

```diff
@@ -46,6 +48,205 @@ <h4>LET’S CONNECT</h4>
+                    <div class="section-frame">
+                        <div class="cannopy">CANOPY
+                            <div class="section-img">
+                                <img src="/app/static/images/Counter.svg" alt="">
+                            </div>
+                        </div>
+                        <div class="backend">Back End Dev
+                            <div class="section-img">
+                                <img src="/app/static/images/Counter.svg" alt="">
+                            </div> 
+                        </div>
+                        <div class="year">2015</div>
+                    </div>
+                    <div class="section-primary-text">
+                        <p>
+                            A daily selection of privately personalized reads; 
+                            no accounts or sign-ups required.
+                        </p>
+                    </div>
+                    <div class="section-tags">
+                        <button type="submit">html</button>
+                        <button type="submit">css</button>
+                        <button type="submit">javascript</button>
+                    </div>
+                    <div class="section-action">
+                        <button type="submit">See project</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+    
+        <!--About me section-->
+        <section class="section-elements">
+            <div class="section-about">
+                <div class="section-block about">
+                    <div class="about-me-text">
+                        <h1>About <br> Myself</h1>
+                    </div>
+                    <div class="about-text">
+                        <p>I’m a software developer! I can help you build a product , 
+                            feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, 
+                            don’t hestiate to contact me.
+                        </p>
+                    </div>
+                    <div class="social-media">
+                        <h4>LET’S CONNECT</h4>
+                        <ul class="icons">
+                            <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                            <li><a href="#"></a><i class="fa-brands fa-linkedin-in"></i></li>
+                            <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+                            <li><a href="#"><i class="fa-brands fa-github"></i></a></li>
+                            <li><a href="#"><img src="/app/static/images/Vector.svg" alt=""></a></li>
+                        </ul>
+                    </div>
+                    <div class="section-action">
+                        <button type="submit">Get my resume</button>
+                    </div>
```

Comment:
- [ ] Kindly check the width set for this button, it is overlapping and the texts are not fitting correctly.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/185554902-05f64bb0-dae6-43d1-9dea-18ff54396204.png)

**This is how it is supposed to be like:**

![image](https://user-images.githubusercontent.com/86318284/185554989-04b5b90d-00bb-4176-bb36-34042b200839.png)